### PR TITLE
Use secure storage plugin in capacitor

### PR DESCRIPTION
**Pull request by yksolanki9**

### CODE REVIEW

File: src/app/core/interceptors/httpInterceptor.ts [hunk 1]
```diff
@@ -68,6 +70,7 @@ export class HttpConfigInterceptor implements HttpInterceptor {
       concatMap((refreshToken) => this.routerAuthService.fetchAccessToken(refreshToken)),
       catchError((error) => {
         this.userEventService.logout();
+        this.secureStorageService.clearAll();
         this.storageService.clearAll();
```
Comment (Dimple16) [inline src/app/core/interceptors/httpInterceptor.ts:74]: will we still need `storageService` here?

File: src/app/auth/switch-org/switch-org.page.ts [hunk 2]
```diff
@@ -213,6 +215,7 @@ export class SwitchOrgPage implements OnInit, AfterViewChecked {
           from(this.proceed()).subscribe(noop);
         },
         async (err) => {
+          await this.secureStorageService.clearAll();
           await this.storageService.clearAll();
```
Comment (Dimple16) [inline src/app/auth/switch-org/switch-org.page.ts:219]: aren't we replacing it?

File: src/app/core/services/secure-storage.service.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      value: JSON.stringify(value),
+    });
+  }
+
+  async get(key: string) {
+    try {
+      const stringifiedObject = await SecureStoragePlugin.get({
+        key,
+      });
+      if (stringifiedObject && stringifiedObject.value) {
```
Comment (Dimple16) [inline src/app/core/services/secure-storage.service.ts:26]: `stringifiedObject?.value`

**Review by Dimple16 — APPROVED** [review]

File: src/app/core/interceptors/httpInterceptor.ts (hunk 1 above)
Comment (yksolanki9) [reply src/app/core/interceptors/httpInterceptor.ts:74]: Yes, we clear all locally stored data when logging out

File: src/app/auth/switch-org/switch-org.page.ts (hunk 2 above)
Comment (yksolanki9) [reply src/app/auth/switch-org/switch-org.page.ts:219]: We clear all cached data on logout and when switching orgs

File: src/app/core/services/secure-storage.service.ts (hunk 3 above)
Comment (yksolanki9) [reply src/app/core/services/secure-storage.service.ts]: Done

**Review by Julias0 — APPROVED** [review]

**Pull request closed without merging**